alpha/lib/model/LiveEntry.php

```diff
@@ -954,6 +955,7 @@ public function getViewMode()
 	public function setViewMode($v)
 	{
 		$this->putInCustomData(self::CUSTOM_DATA_VIEW_MODE, $v);
+		$this->setLastChangedViewMode(time());
```

Comment:
i think you need to call setLastChangedViewMode only when it actually changed and not the same..